schema/algolia-reverse-timestamp-schema.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Mateusz: maybe it is worth to put attributesForFaceting and attributesToIndex into one place?

Reply:
@niezgoda not sure I follow here.